staff_features/accessions/accession_transfer.feature

```diff
@@ -1,13 +1,13 @@
-Feature: Accession transfer to another Repository
+Feature: Accession Transfer
   Background:
    Given an administrator user is logged in
+     And a Repository with name 'Transfer Test Repository' has been created
      And an Accession has been created
-     And a Repository has been created
      And the Accession is opened in edit mode
   Scenario: Accession is transferred to another Repository
-    When the user clicks on 'Transfer' button on the toolbar
-     And the user selects the destination Repository from the drop-down list
-     And the user clicks on 'Transfer' button
-     And the user clicks on 'Transfer' button at the modal
+    When the user clicks on 'Transfer'
+     And the user selects 'Transfer Test Repository' from 'Destination Repository'
+     And the user clicks on 'Transfer' in the dropdown menu
```

Comment:
I think it is more accurate instead of 'dropdown menu' to use 'transfer form' as there is not any obvious drop down menu here.

Reply:
I did these on purpose since they work out of the box with the current step definitions, and the end goal is to have as much simplicity and re-usability as possible. Even though it's technically a form, everything that has a `caret` symbol in the system is a dropdown menu. From a naive future user perspective, it will be intuitive to write `in the dropdown` for anything included inside a dropdown.

I have updated this one to use `in the transfer form`.